tests/testthat/test-transition_risk_profile.R

```diff
@@ -103,3 +103,56 @@ test_that("outputs `NA` transition risk category for `NA` transition risk score
   # `transition_risk_category` is `NA` for `NA` transition risk score
   expect_true(is.na(unique(output$transition_risk_category)))
 })
+
+test_that("outputs columns `transition_risk_category_share` and `transition_risk_category` at company level", {
```

Comment:
A few things:

* This test doesn't seem to need a seed. We're only testing for column names -- not their content.
* The setup is really large and burries the intention of the test.
* Are we planning more tests? The fact that the output has a specific column doesn't say much about whether or not the values of that column are correct. 
* What reasaonable edge cases do w need to test?

Reply:
@maurolepore The edge cases are same for `add_transition_risk_category_at_company_level` function because it uses the function `epa_at_company_level()` from tiltIndicator package. I can see a lot tests for this function in tiltIndicator which covers possible edge case. Do you want me to copy all those test cases here as well?

Reply:
Mmm it shouldn't be about what I "want". Please take my reviews as suggestions intended to spark ideas about how to improve the code. As the author of this function and as the maintainer of the package, you are the one with the deepest context and you're welcome to take or leave my ideas -- which due to my superficial understanding may well be off.

Reply:
@maurolepore For example, we have column A and column B which have similar unique values but belongs to different tables, then should I test the column A if we have good amount of tests for column B? Is it a good practice to have similar tests for column A as well?

Reply:
Ticket: https://github.com/2DegreesInvesting/tiltIndicatorAfter/issues/267